nano/templates/shuttle_mainpulator.tmpl.txt

```diff
@@ -0,0 +1,133 @@
+<ui-tabs tabs='{{data.tabs}}'>
```

Comment:
OH SHIT you know what...this isn't supposed to be there file wise @Crazylemon64 i am not using nano...i am using just that datum browser code! I WILL REMOVE THIS FILE NEXT COMMIT....cuase this is the TGUI code..and unless someone can tell me how to go from TG UI back to NANO i will use the datum browser code..